test/vcf/predefined_info_tags_test.cpp

```diff
@@ -42,12 +42,12 @@ namespace ebi
                             "chr1",
                             123456,
                             { "id123" },
-                            "A",
+                            vcf::A,
```

Comment:
same as predefined_format_tags_test, this usages (bases string) should not be extracted as the same constant for "cardinality A". revert all usages in this file, please